pkgs/tools/misc/farge/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bash
+, bc
+, feh
+, grim
+, imagemagick
+, slurp
+, wl-clipboard
+, xcolor
+, waylandSupport ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "farge";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "sdushantha";
+    repo = "farge";
+    rev = "v${version}";
+    hash = "sha256-rTY4vQkrcyVRfIHt1qtFpxaC5bmpJ2/C6NVhEDkxef8=";
+  };
+
+  buildInputs =
```

Comment:
Please extract the common part and do something like:

```nix
buildinputs = [ bash bc ] ++ lib.optionals waylandSupport [ slurp wl-clipboard ];
```

Reply:
Okay, done